config/routes.rb

```diff
@@ -21,7 +21,9 @@
   namespace :api, defaults: { format: :json } do
     namespace :v1 do
       namespace :admin do
-        resources :playlist_activities, only: [:update]
+        resources :students, only: [:update] do
+          patch '/playlist_activities/:id', controller: 'students/playlist_activities', action: 'update'
+        end
       end
```

Comment:
I'm also not sure why you are passing the :id at the end. are you updating one playlist activity at a time, or the whole set? it seems like from the way you've written the controller, you expect to patch all the playlist activities at once?